### PR TITLE
Code Review Week 1: development of  review protocol

### PR DESCRIPTION
**Pull request by Zcalory**
According to Kitchenham's method, this week we develop a review protocol that will be used to finish our survey. Please check it and give us some suggestions.

### CODE REVIEW

File: 05-code_review_protocol.Rmd [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# (APPENDIX) Appendix {-}
```
Comment (nsalminen) [inline 05-code_review_protocol.Rmd]: Is there a specific reason for why the first line of your chapter has an “Appendix” header?

File: 05-code_review_protocol.Rmd [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+# Code Review protocol
+This appendix describes the review protocol used for the systematic review presented in the
+section on *Code Review*. The protcol has been set up using Kitchenham's method as described
```
Comment (nsalminen) [inline 05-code_review_protocol.Rmd]: prtocol -> protocol

File: 05-code_review_protocol.Rmd [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+in @kitchenham2007.
+
+## Background
+
+## Research questions
+The goal of the review is to summarize the state of the art and identify future challenges in the
+code review area. The research questions are as follows:
+
+* What is the state of the art in the research area of code review?
+* What is the expectations and outcomes of the code review? Do they match and What is the difference?
```
Comment (nsalminen) [inline 05-code_review_protocol.Rmd]: You capitalized the second “What” here.

File: 05-code_review_protocol.Rmd [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+* What are future challenges in the research area of code review?
+
+## Search process
+Given six initial paper seed to have a basic understanding about this topic, we then search for more paper to obtain  more information to complete the survey. The search process consits of the following:
```
Comment (nsalminen) [inline 05-code_review_protocol.Rmd]: It may be better to split the first sentence up in two and rewrite them a bit. The formulation is a bit unclear.

Comment (nsalminen) [reply 05-code_review_protocol.Rmd]: “consits” -> “consists”

File: 05-code_review_protocol.Rmd [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+## Quality assessment
+
+## Data collection
+The following data will be collected from each considered resource:
+
+* Source (for example, the blog website or specific journal)
+* Year published
+* Type of resource
+* Sub-area of code review
```
Comment (nsalminen) [inline 05-code_review_protocol.Rmd]: Which “sub-areas of code review” do you consider?

File: 05-code_review_protocol.Rmd [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+* Author(s) and organization(s)
+* Research question(s), if applicable
+* Summary of the resource. In case of scientific literature: include the main research questions and conclusions.
+* Quality score
+
+All data will be collected by one person participating in the review and checked by another.
```
Comment (nsalminen) [inline 05-code_review_protocol.Rmd]: Why did you choose for the strategy of having only one of your contributors conduct the search for literature?

File: 05-code_review_protocol.Rmd [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+## Data analysis
+
+## Dissemination
+
+### Next step
+The next plan is to collect a lot of paper to obtain deeper understanding aiming at solving the research questions and have a clear standard of quality assessment.
```
Comment (nsalminen) [inline 05-code_review_protocol.Rmd]: It's best to split up sentences like this, in this case we would suggest something along the lines of: “We plan to collect papers in order to gain understanding of the topic and answer our RQs. Furthermore, we plan to come up with a clear standard for quality assessment.”

File: 05-code_review_protocol.Rmd (hunk 3 above)
Comment (lsikkes) [inline 05-code_review_protocol.Rmd]: What "are" the expectations

**Comment by lsikkes** [comment]
I'm missing the found papers that will be used. Which ones have you found? How have you found them? And why will you use these?

**Pull request closed without merging**